### DISCUSSION TITLE
Mangal 3 Released!

### DISCUSSION THREAD

**Discussion by metafates** [metafates/mangal#64] [Announcements]
After a long break Mangal 3 was finally released! This is a complete rewrite from scratch. This update brings a lot of changes, fixes old bugs, but most importantly now Mangal supports Lua based scrapers. See README for more information. Feel free to leave your feedback here!

**Comment by OmiChrome** [comment]
add the scoop update ASAP :)

**Reply by metafates** [reply]
Scoop should be working. Did you try this?

```
scoop bucket add metafates
scoop install mangal
```

**Reply by OmiChrome** [reply]
yes  i have installed it already . 
i am only saying for update the package for scoop :)

**Reply by OmiChrome** [reply]
oh wait ?

**Reply by OmiChrome** [reply]
![image](https://user-images.githubusercontent.com/108952834/183501138-46fd1dd5-2a09-4fc5-8399-dac74b19254c.png)
your bucket is not correct

**Reply by metafates** [reply]
ughh, thanks for pointing that out. Can you please try that?

```
scoop bucket add metafates https://github.com/metafates/scoop-metafates
scoop install mangal
```

**Reply by OmiChrome** [reply]
bro thx :D it worked 

also update that in readme.md  too

**Reply by metafates** [reply]
Sure, thank you!

**Reply by justchokingaround** [reply]
i came

**Comment by OmiChrome** [comment]
hey
 what is darwin  in release area ?

**Reply by metafates** [reply]
Long story short - it's macOS

https://apple.stackexchange.com/questions/401832/why-is-macos-often-referred-to-as-darwin
